overlays/dp1/cnpg-butler-dp1-backup.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: postgresql.cnpg.io/v1
+kind: ScheduledBackup
+metadata:
+  name: usdf-butler-dp1-backup
+  namespace: dp1
+spec:
+  immediate: true
+  schedule: "0 0 0 * * *"
```

Comment:
We should stagger the times so S3 doesn't get hit at once from all clusters with backups.

Reply:
Noon EST is good, but have them at 12:02, 12:05, 12:08, etc. rather than all at 12:00.  `cron` has a `RANDOM_DELAY` variable that can also be used to do this.